ngraph/test/frontend/paddlepaddle/op.cpp

```diff
@@ -0,0 +1,203 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <regex>
+#include <fstream>
+
+#include "util/all_close.hpp"
+#include "util/all_close_f.hpp"
+#include "util/engine/test_engines.hpp"
+#include "util/ndarray.hpp"
+#include "util/test_case.hpp"
+#include "util/test_control.hpp"
+#include "util/test_tools.hpp"
+
+#include "ngraph/ngraph.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
+
+#include "../shared/include/basic_api.hpp"
+
+// library taken from https://github.com/llohse/libnpy
```

Comment:
It doesn't look acceptable to take header file from other repo and change copyright notice.

Consider some another approach to download and build dependencies, like it is done for 'protobuf' (however may still need some legal review)

Reply:
@ceciliapang2011, could you help refine this ?